openmc/plots.py

```diff
@@ -382,7 +460,7 @@ def get_plot_xml(self):
         subelement = ET.SubElement(element, "pixels")
         subelement.text = ' '.join(map(str, self._pixels))
 
-        if self._mask_background is not None:
+        if self._background is not None:
```

Comment:
Nice catch! It looks like this is actually what is causing a few tests to fail. Can you change the default value in the constructor to:
```Python
self._background = None
```
That should fix the failing tests.